code/modules/tables/interactions.dm

```diff
@@ -135,7 +135,7 @@
 		user << SPAN_WARNING("There's nothing to put \the [W] on! Try adding plating to \the [src] first.")
 		return
 
-	if (user.unEquip(W, src.loc))
-		set_pixel_click_offset(W, params)
+	set_pixel_click_offset(W, params)
+	user.unEquip(W, src.loc)
```

Comment:
nooop

this is a safety check. unEquip returns true if it succeeds, and THEN we move the object
these two should be restored to how they were

Reply:
There are no cases in this very situation, when it will return FALSE. Prove me wrong.

Reply:
(P.S.: And even if it does, the item will stay in hand, and there it doesn't matter what it's pixel_x and pixel_y is).